modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -510,3 +510,48 @@
 	icon_state = "onion"
 	item_state = "onion"
 	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+
+/obj/item/clothing/under/mikubikini
+	name = "starlight singer bikini"
+	desc = " "
+	icon_state = "mikubikini"
+	item_state = "mikubikini"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/suit/mikujacket
+	name = "starlight singer jacket"
+	desc = " "
+	icon_state = "mikujacket"
+	item_state = "mikujacket"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/head/mikuhair
+	name = "starlight singer hair"
+	desc = " "
+	icon_state = "mikuhair"
+	item_state = "mikuhair"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
```

Comment:
Add in a `flags_inv = HIDEHAIR` or the wearer's haircut will probably be also shown...
Not that this wig doesn't stick out like a sore thumb already.

Reply:
flag**s**_inv, not flag_inv.